modules/post/windows/gather/enum_files.rb

```diff
@@ -75,44 +74,47 @@ def download_files(location, file_type)
       filename = "#{file['path']}\\#{file['name']}"
       data = read_file(filename)
       print_status("Downloading #{file['path']}\\#{file['name']}")
-      p = store_loot("host.files", 'application/octet-stream', session, data, file['name'], filename)
+      p = store_loot("host.files", 'application/octet-stream', session, data, file['name'], filename, nil, true)
       print_good("#{file['name']} saved as: #{p}")
     end
   end
 
 
   def run
     # When the location is set, make sure we have a valid path format
-    location = datastore['SEARCH_FROM']
-    if location and location !~ /^([a-z])\:[\\|\/].*/i
-      print_error("Invalid SEARCH_FROM option: #{location}")
-      return
-    end
+    datastore['SEARCH_FROM'].split(",").each do |location|
```

Comment:
This snippet of code would lead to unexpected error, as below shows:
```
msf5 post(windows/gather/enum_files) > run

[-] Post failed: NoMethodError undefined method `split' for nil:NilClass
[-] Call stack:
[-]   /Users/green/msfdev/metasploit-framework/modules/post/windows/gather/enum_files.rb:85:in `run'
[*] Post module execution completed
```